airframe-http/.jvm/src/main/scala/wvlet/airframe/http/router/HttpResponseCodec.scala

```diff
@@ -23,8 +23,8 @@ class HttpResponseCodec[Resp: HttpResponseAdapter] extends MessageCodec[HttpResp
   override def pack(p: Packer, v: HttpResponse[_]): Unit = {
     v.contentType match {
       case Some("application/x-msgpack") =>
+        // Raw msgpack response
```

Comment:
This code path has never been used in our application unless the API returns raw HTTP Response class with msgpack body content.